backend/package.json

```diff
@@ -0,0 +1,20 @@
+{
+    "scripts": {
+        "start": "cd src && node server.js"
+    },
+    "devDependencies": {
+        "axios": "^0.16.2",
```

Comment:
please remind me, why do you need axios in dev dependencies here?

Reply:
oh, many modules in many directories..
axios must not be into backend

Reply:
why didn't you remove it then before merging pull request?

Reply:
fixed in https://github.com/emotz/kladovka/commit/81e34dcf8f27da71efe6d27dc9474c1da5a96007